src/components/buttons/z-storia-button/styles.css

```diff
@@ -0,0 +1,143 @@
+::slotted(button), ::slotted(a) {
+  font-family: var(--dashboard-font);
+  font-weight: var(--font-sb);
+  font-size: 14px;
+  letter-spacing: 0.3px;
+  border-width: var(--border-size-medium);
+  border-style: solid;
+  border-radius: var(--border-radius);
+  height: 44px;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: 0px calc(var(--space-unit) * 2);
+  cursor: pointer;
+  white-space: nowrap;
+  outline: none;
+  width: inherit;
+}
+
+::slotted(button)::-moz-focus-inner , ::slotted(a)::-moz-focus-inner {
+  border: 0;
+}
+
+::slotted(button).primary, ::slotted(a).primary {
```

Comment:
We don't need to use classes, this kind of selector should be better: `:host([variant="secondary"]) ::slotted(button)`

Reply:
Correct. I should have fixed that.